src/utils/transformChunk.ts

```diff
@@ -0,0 +1,67 @@
+import { decode } from 'sourcemap-codec';
+import error from './error';
+import { OutputOptions, RawSourceMap } from '../rollup/types';
+import Graph from '../Graph';
+import Chunk from '../Chunk';
+
+export default function transformChunk(
+	graph: Graph,
+	chunk: Chunk,
+	code: string,
+	sourcemapChain: RawSourceMap[],
+	options: OutputOptions
+) {
+	return graph.plugins.reduce((promise, plugin) => {
+		if (!plugin.transformBundle) return promise;
```

Comment:
Actually, this might be the reason why `transformChunk` does not work at all...

Reply:
Ahh, thanks.